### ISSUE TITLE
play_tone function should not raise Value error on frequency of 0

### ISSUE THREAD

**Issue by RufusVS** [adafruit/Adafruit_CircuitPython_MagTag#67]
Currently in the library, if a zero frequency is specified to play_tone, a ValueError is thrown.   In most other similar implementations, a value of 0 simply represents silence, or rest, for the duration specified.  There is no reason for this implementation to be different.  Also, this change is not likely to affect any user's existing code.

Current Code:
```
def play_tone(frequency, duration):
        """..."""
        if frequency <= 0:
            raise ValueError("The frequency has to be greater than 0.")
```

Suggested change:
```
def play_tone(frequency, duration):
        """..."""
        if frequency < 0:
            raise ValueError("Negative frequencies are not allowed.")
```

**Comment by tannewt** [comment]
@kattni What do you think the right way of doing rests is?

**Comment by tannewt** [comment]
`play_tone` is blocking so it could just be a `time.sleep(duration)` instead.

**Comment by tannewt** [comment]
I agree that in this case it is simpler. However, I'm not sure that it's better in the case where frequency is the result of a computation that shouldn't return zero but does due to an error. Input validation can make it easier to find bugs. I'm not sure allowing zero frequency is better for all cases.

**Comment by makermelissa** [comment]
Fixed via #68.
